examples/user_guide/Links.ipynb

```diff
@@ -432,7 +453,7 @@
    ]
   }
  ],
- "metadata": {
+  "metadata": {
```

Comment:
```suggestion
 "metadata": {
```

(not sure how that stray space got there)

Reply:
@hoseppan could you apply this fix?